app/pages/admin/project-status.jsx

```diff
@@ -164,6 +210,8 @@ class ProjectStatus extends Component {
         <div className="project-status__section">
           <h4>Workflow Settings</h4>
           <small>The workflow level dropdown is for the workflow assignment experimental feature.</small>
+          <br />
+          <small>An asterisk (*) denotes a default workflow.</small>
           {this.renderError()}
```

Comment:
There's a React error being thrown, `Error: Objects are not valid as a React child` when an error is set to the component because this is expecting a `this.state.error` to be a string. We changed how errors are handled in the javascript client to send along the full error object instead, so how the error renders should be modified too. Perhaps the status and message together?:
```
{`Error ${this.state.error.status}: ${this.state.error.message}`}
```